_posts/en/newsletters/2025-05-16-newsletter.md

```diff
@@ -0,0 +1,150 @@
+  in scenarios involving very large channels or channels that don't fit a
+  bimodal distribution. Additionally, the model now skips unnecessary
+  probability calculations and automatically corrects outdated mission control
+  data and contradictory historical information.
+
+- [Rust Bitcoin #4458][] replaces the `MtpAndHeight` struct with an explicit
+  pair of the newly added `BlockMtp` and the already existing `BlockHeight`,
+  enabling better modeling of both block height and Median Time Past (MTP)
+  values in relative [timelocks][topic timelocks]. Unlike
+  `locktime::absolute::Mtp`, which is constrained to values above 500 million
+  (roughly after 1985), `BlockMtp` can represent any 32-bit timestamp. This
+  makes it suitable for theoretical edge cases, such as chains with unusual
+  timestamps. This update also introduces `BlockMtpInterval`, and renames
+  `BlockInterval` to `BlockHeightInterval`.
+
+- [BIPs #1848][] updates the status of [BIP345][] to `Withdrawn`, as the author
+  believes its proposed `OP_VAULT` opcode has been superseded by
+  [`OP_CHECKCONTRACTVERIFY`][topic matt] (OP_CCV), a more general vault design
+  and a new type of [covenant][topic covenants].
+
+- [BIPs #1841][] merges [BIP172][], which proposes formally defining Bitcoin’s
+  indivisible base unit as a “satoshi,” reflecting current widespread usage and
+  helping standardize terminology across applications and documentation.
+
+- [BIPs #1821][] merges [BIP177][], which proposes redefining “bitcoin” to
+  represent the smallest indivisible unit (commonly referred as 1 satoshi),
+  rather than 100,000,000 units. The proposal argues that aligning terminology
+  with the actual base unit would reduce confusion caused by arbitrary decimal
+  conventions.
+
+{% include snippets/recap-ad.md when="2025-05-20 16:30" %}
+{% include references.md %}
+{% include linkers/issues.md v=2 issues="32155,28710,8272,8330,4458,1848,1841,1821" %}
+[lnd 0.19.0-beta.rc4]: https://github.com/lightningnetwork/lnd/releases/tag/v0.19.0-beta.rc4
+[news314 excess addr]: /en/newsletters/2024/08/02/#remote-crash-by-sending-excessive-addr-messages
+[stewart bippost]: https://groups.google.com/g/bitcoindev/c/j1zEky-3QEE
+[64bit bip]: https://github.com/Christewart/bips/blob/2025-03-17-64bit-pt2/bip-XXXX.mediawiki
+[news285 64bit]: /en/newsletters/2024/01/17/#proposal-for-64-bit-arithmetic-soft-fork
+[news290 64bit]: /en/newsletters/2024/02/21/#continued-discussion-about-64-bit-arithmetic-and-op-inout-amount-opcode
+[news306 64bit]: /en/newsletters/2024/06/07/#updates-to-proposed-soft-fork-for-64-bit-arithmetic
+[stewart inout]: https://delvingbitcoin.org/t/op-inout-amount/549/4
+[stewart overflow]: https://delvingbitcoin.org/t/overflow-handling-in-script/1549
+[poinsot addrvuln]: https://mailing-list.bitcoindevs.xyz/bitcoindev/EYvwAFPNEfsQ8cVwiK-8v6ovJU43Vy-ylARiDQ_1XBXAgg_ZqWIpB6m51fAIRtI-rfTmMGvGLrOe5Utl5y9uaHySELpya2ojC7yGsXnP90s=@protonmail.com/
```

Comment:
we use bitcoindevs.xyz here for the ML but the google group link for [stewart bippost]

Reply:
bitcoindevs.xyz isn't updating and doesn't have the Stewart post, so I had to use Google Groups for that link.  :-(